src/sentry/integrations/slack/requests/action.py

```diff
@@ -46,6 +46,8 @@ def callback_data(self) -> JSONData:
         if self.data["type"] == "block_actions":
             if self.data.get("view"):
                 return json.loads(self.data["view"]["private_metadata"])
+            elif self.data["container"].get("is_app_unfurl"):
+                return json.loads(self.data["app_unfurl"]["blocks"][0]["block_id"])
```

Comment:
@isabellaenriquez feels like we should check for an empty array here so this doesn't blow up

Reply:
you mean just checking that there are blocks in `"app_unfurl"`?

Reply:
that and `data["app_unfurl"]["blocks"]` is an array of at least size 1